src/order/antichain.lean

```diff
@@ -94,6 +94,35 @@ lemma insert_of_symmetric (hs : is_antichain r s) (hr : symmetric r)
   is_antichain r (insert a s) :=
 (is_antichain_insert_of_symmetric hr).2 ⟨hs, h⟩
 
+lemma image_rel_iso (hs : is_antichain r s) (φ : rel_iso r r') :
+  is_antichain r' (φ '' s) :=
```

Comment:
Can you also add the `iff` version?
Is a `rel_embedding` enough for one of those implications?
Same for the lemmas below.

Reply:
Thanks - it all holds (with implications in both directions) at the level of embeddings. Do you think it is worth even leaving the `iso`/`rel_iso` versions in, given that there is a coercion?

Reply:
The latest commit has all the variants.